sdk/src/azure/core/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ add_library (
   ${CMAKE_CURRENT_LIST_DIR}/az_base64.c
   ${CMAKE_CURRENT_LIST_DIR}/az_context.c
   ${CMAKE_CURRENT_LIST_DIR}/az_hfsm.c
+  ${CMAKE_CURRENT_LIST_DIR}/az_hfsm_pipeline.c
```

Comment:
Should this be named pipeline_internal.c?

Reply:
I am not certain, none of the other files have _internal as a suffix even if they do have an internal .h (they do have a non-internal.h though).